docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,306 @@
+---
+title: "AWS Route 53 GSLB Multi-Region Proxy Peering High Availability Deployment Guide"
+description: "Deploying a Proxy-peered High Availability Teleport Cluster using Route 53 to create Global Server Load Balancing"
+---
+
+When deploying Teleport in production, you should design your deployment to
+ensure that users can continue to access infrastructure should an outage or 
+incident affect the availability of your Teleport cluster. 
+
+In order to maintain optimal end-user 
+experience with minimal latency and maximum performance, it is imperative to ensure the scalability 
+of your Auth Service and Proxy Service to accommodate increasing numbers of users and connected resources.
+
+(!docs/pages/includes/cloud/call-to-action.mdx!)
+
+## Overview
+This deployment architecure makes all connected resources accessible through a single Teleport cluster 
+across multiple regions using exclusively AWS ecosystem infrastructure. 
+
+This is accomplished using AWS Route 53 to create Global Server Load Balancing (GSLB) 
```

Comment:
I might rephrase this a bit to make it clear what these two pieces provide. Maybe something like this:

This deployment architecture features two important design decisions:
1. AWS Route 53 is used for Global Server Load Balancing (GSLB). This allows for efficient distribution of traffic across resources that are globally distributed.
2. Teleport's Proxy Peering is used to reduce the total number of tunnel connections in the Teleport cluster.

Reply:
I like your recommendations. The only change I made was specifying Latency based routing is being used as I think readers could easily assume Geolocation routing is being used. It is worth repeating through the guide in my opinion because Latency based is going to improve performance whereas Geolocation won't (as much).